src/CheckboxInput.js

```diff
@@ -1,19 +1,24 @@
-import React, { forwardRef } from 'react'
+import React from 'react'
 import styled from 'styled-components/macro'
 
-const CheckboxInput = forwardRef(({ question }, ref) => {
+const CheckboxInput = ({ question, register, watch }) => {
   return (
     <div>
       {question.answerOptions.map((answerOption, innerIndex) => {
```

Comment:
innerIndex? Its just the index isn't it?

Reply:
Thanks, I fixed it :-)